docker/build/static_libs.patch

```diff
@@ -6,93 +6,57 @@ index 00fb3a632..923739d2b 100644
 --- a/src/bin_client/dune
```

Comment:
Isn't it possible to use dune static profile now?
AFAIR, we decided to avoid using it back in the days when `10.*` didn't have it and `11.*` had it

Reply:
Hmm, in fact, it is already used: https://github.com/serokell/tezos-packaging/blob/master/docker/build/build-tezos.sh#L24. Can you check if these patches are now actually needed for all binaries (except for the `tezos-sandbox`)?